apidoc/docgen.js

```diff
@@ -455,7 +455,16 @@ function processAPIs (api) {
 					'__subtype': 'method'
 				};
 				api.__creatable = true;
-				'methods' in doc[cls] ? doc[cls].methods.push(createMethod) : doc[cls].methods = [createMethod];
+				if ('methods' in doc[cls]) {
+					if (!doc[cls].methods) {
```

Comment:
What's the difference between the `'methods' in doc[cls]` and the `doc[cls].methods` statement? Don't they both check if the `methods` key is included? I mean: It should be fine if no `methods` tag is specified, it shoud rather warn if there is a tag specified but that one is empty. Or is it that what it does?